web/wp-content/themes/dctx-v2/footer.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-wgqlxh.php, but found phpcs-scan-WGqlxH.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).